qbft/state.go

```diff
@@ -5,6 +5,8 @@ import (
 )
 
 type IConfig interface {
+	// GetSignatureDomainType returns the Domain type used for signatures
+	GetSignatureDomainType() types.DomainType
```

Comment:
Why this function has been added in `IConfig`? I see that the `Config` structure also has it but it seems unused.

If that's the case we can actually drop it all. Maybe in another PR since it's not the scope of this PR. If you agree, pls create an issue so we can solve this later :)

Reply:
You are right, dropping this